manifests/service.pp

```diff
@@ -55,7 +55,22 @@
       enable => true,
     }
 
-  } else {
+  } elsif ($::operatingsystem == 'RedHat') or ($::operatingsystem == 'CentOS') {
```

Comment:
Only CentOS 7 uses systemd, this needs to check the version. The previous if-statement already handles systemd for ubuntu/debian, so you can add these conditions to the "or"